README.md

```diff
@@ -14,6 +14,9 @@ Use [Composer](https://getcomposer.org/) to install the package:
 $ composer require openeuropa/cdt-client
 ```
 
+***Note**: `cweagans/composer-patches` should be removed from the allowed plugin list after 
```

Comment:
I wasnt clear enough, I means in the readme section of composer (this doesn't concern the reader of this file)